app/views/projects/show.api.rsb

```diff
@@ -14,6 +14,10 @@ api.project do
     api.geojson ""
   end
 
+  if @project.gtt_tile_sources.present?
+    api.gtt_tile_sources @project.gtt_tile_sources
+  end
+
```

Comment:
@nobnisai @dkastl (CC: @mopinfish, @smellman)
About exposing tile sources information, let me confirm this more from other point of view. 🙇‍♂️